Weather Alert.xcodeproj/project.pbxproj.orig

```diff
@@ -0,0 +1,553 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 46;
+	objects = {
+
+/* Begin PBXBuildFile section */
+<<<<<<< HEAD
```

Comment:
You haven't resolved merge conflict correctly, this file won't run with  this <<<<<<< HEAD ======= >>>>>>>